packages/ui-core/src/components/slider/utils/colors.utils.ts

```diff
@@ -1,4 +1,7 @@
-export const stringifyColors = (colors: string[], colorSteps: number) => {
-  const newColors = colors.slice(0, colorSteps);
+export const stringifyColors = (colors: string[], zeroPoint: number) => {
+  const newColors = colors.map((color) => {
+    if (color === '#FFFFFF') return `${color} ${zeroPoint}%`;
```

Comment:
What for use case when color equals `#fff` ? Maybe we need to create additional handler like `isWhiteColor`.

Reply:
`color` here is one from our palette, so I can check if it equal `colors.white[500]`(this one is added here when min < 0 && max > 0). Then if our white will be changed into: `#fff` it will still be valid.

Reply:
👍